python/pyspark/pandas/tests/test_expanding.py

```diff
@@ -97,16 +97,22 @@ def test_expanding_skew(self):
     def test_expanding_kurt(self):
         self._test_expanding_func("kurt")
 
-    def _test_groupby_expanding_func(self, f):
+    def _test_groupby_expanding_func(self, ps_func, pd_func=None):
+        if not pd_func:
+            pd_func = ps_func
```

Comment:
Sorry, I got a bit confusion for this part.

Could you tell one example that considers this case?

Reply:
I think it's that it uses the same function if `pd_func` is omitted.

Reply:
Yes, just like: `self._test_groupby_rolling_func("count")`.

We add `ps_func` and `pd_func` separately for some case like: `self._test_groupby_expanding_func(lambda x:x.quantile(0.5), lambda x: x.quantile(0.5, "lower"))`, `pd_func` and `ps_func` is different.